lightning/src/routing/router.rs

```diff
@@ -1476,6 +1476,21 @@ where L::Target: Logger {
 	}
 
 	// Step (5).
+	// We only consider paths shorter than our maximum length estimate. 
```

Comment:
Limiting the path length could also be done during Step (3) / `path_construction`. I however somehow consider it more a path selection logic, which is why I put it here.

Reply:
Alright, then probably the way to go is to _only_ do it there, no? Because if we exclude any longer paths from the get-go, we should at this point never have to deal with longer paths at all?

Reply:
Moved it to path walking now with https://github.com/lightningdevkit/rust-lightning/pull/1476/commits/1098208367cf995178da752d7057832419a60944.